index.js

```diff
@@ -2,11 +2,19 @@
 
 const fp = require('fastify-plugin')
 const assert = require('assert')
+const { monitorEventLoopDelay } = require('perf_hooks')
+
+function getSampleInterval (value, eventLoopResolution) {
+  const defaultValue = monitorEventLoopDelay ? 1000 : 5
```

Comment:
I think we should document this in the README and we are good to go!

Reply:
Ok 🎉